index.html

```diff
@@ -4,8 +4,70 @@
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Heloo cursera</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css" >
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+
+
+	<script>
+
+	function load_home() {
+
+		$(document).ready(function(){
+			$('.container').load("/menu/category.html");
+
+		});
+	}
+
+   function openMenu() {
+		var x = document.getElementById("myLinks");
+		if (x.style.display === "block") {
+		x.style.display = "none";
+	} else {
+		x.style.display = "block";
```

Comment:
it is better to use const for comparing string, especially if you write it more the one time.